app/app.js

```diff
@@ -34,7 +34,7 @@ openSansObserver.check().then(() => {
 });
 
 // Import the CSS reset, which HtmlWebpackPlugin transfers to the build folder
-import '../node_modules/sanitize.css/dist/sanitize.min.css';
+import '../node_modules/sanitize.css/sanitize.css';
```

Comment:
Hey just woke up. I think npm is installing a newer version of sanitize, where the dist folder doesn't exist anymore. So there's not a minified version of it included anymore. Clone the project and do a fresh npm install to see.